packages/gatsby-source-prismic/src/builders/buildImageBaseFieldConfigMap.ts

```diff
@@ -1,76 +1,62 @@
-import * as gqlc from 'graphql-compose'
-import * as gatsbyFs from 'gatsby-source-filesystem'
-import * as gatsbyImgix from 'gatsby-plugin-imgix/dist/node'
-import * as RTE from 'fp-ts/ReaderTaskEither'
+import * as imgixGatsby from '@imgix/gatsby/dist/pluginHelpers'
 import { pipe } from 'fp-ts/function'
-
+import * as RTE from 'fp-ts/ReaderTaskEither'
+import * as gatsbyFs from 'gatsby-source-filesystem'
+import * as gqlc from 'graphql-compose'
 import { Dependencies, PrismicAPIImageField } from '../types'
 
 const resolveUrl = (source: PrismicAPIImageField): string | null => source.url
 
-const resolveWidth = (source: PrismicAPIImageField): number | undefined =>
+/* const resolveWidth = (source: PrismicAPIImageField): number | undefined =>
   source.dimensions?.width
 
 const resolveHeight = (source: PrismicAPIImageField): number | undefined =>
-  source.dimensions?.height
+  source.dimensions?.height */
```

Comment:
The `resolveWidth` and `resolveHeight` functions would be great to have part of creating the fields. Without these functions, the Imgix plugin will need to make a network request for each image to gather width/height metadata to compute the aspect ratio via the `fm=json` Imgix option.

To be more direct of its usage, it could be a single `resolveAspectRatio` function.

`gatsby-plugin-imgix` uses this function to fetch the aspect ratio if the width/height resolve functions return invalid values: https://github.com/WalltoWall/gatsby-plugin-imgix/blob/14592b6eb4678f61f98491e745b477e244adbf4e/src/shared.ts#L54

Reply:
Yep this shouldn't be a problem, I have been working on this in a separate branch and will push this up shortly.

Reply:
Awesome!